SPECS/python3-geomet/python3-geomet.spec

```diff
@@ -0,0 +1,53 @@
+Name:           python3-geomet
+Version:        0.2.1
+Release:        1%{?dist}
+Summary:        GeoJSON <-> WKT/WKB conversion utilities
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages/Python
+Url:            https://github.com/geomet
+Source0:        https://github.com/geomet/geomet/archive/refs/tags/%{version}.tar.gz#/geomet-%{version}.tar.gz
+BuildRequires:  python3
+BuildRequires:  python3-libs
+BuildRequires:  python3-setuptools
+Requires:       python3
+Requires:       python3-libs
+Requires:       python3-six
+Requires:       python3-click
```

Comment:
`python3-click` is in the SPECS-EXTENDED folder. We don't allow specs in the SPECS folder to depend on anything in SPECS-EXTENDED. Please promote the `python-click` package to the SPECS folder.

Reply:
Already done: https://github.com/microsoft/CBL-Mariner/pull/3319